src/app.js

```diff
@@ -231,6 +293,32 @@ export const start = async args => {
     screen.render();
   });
 
+  branchTable.key('n', () => {
+    // @ts-ignore
```

Comment:
Are the ts-ignore comments necessary if we're not using a TS compiler and/or we don't have a ts-linter?